extensions/ql-vscode/src/stories/variant-analysis/VariantAnalysisAnalyzedRepos.stories.tsx

```diff
@@ -125,22 +126,19 @@ Example.args = {
 };
 
 faker.seed(42);
-const uniqueNumbers = [...Array(1000).keys()].sort(() => Math.random() - 0.5);
 
 const manyScannedRepos = Array.from({ length: 1000 }, (_, i) => {
   const mockedScannedRepo = createMockScannedRepo();
-  const randomInt = uniqueNumbers.pop();
-  const uniqueId =
-    randomInt === undefined ? Math.random() * 8000 + 1001 : randomInt;
-
+  const nanoid = customAlphabet("0123456789");
   return {
     ...mockedScannedRepo,
     analysisStatus: VariantAnalysisRepoStatus.Succeeded,
     resultCount: faker.number.int({ min: 0, max: 1000 }),
     repository: {
       ...mockedScannedRepo.repository,
-      id: uniqueId,
-      fullName: `octodemo/${uniqueId}`,
+      // We need to ensure the ID is unique for React keys
+      id: parseInt(nanoid()),
```

Comment:
Is `parseInt` a left over from before? `nanoid()` gives you back a string, not a number.

Reply:
Yes, that's why I have to convert it to a number for the id. Nanoid has no method that returns a number, however by giving it the custom alphabet we can parse the string back to a number.

Reply:
Ok I'm now following a bit more what we're trying to do 😂 I didn't realise this is an id for a repo (which is expected to be a number). Although I'm still unclear why we aren't just using `Math.random()` 🤔 

I think what you have is fine and I think it'll work because there are enough characters to avoid collisions. This is also storybook code so it's not terrible if things break 😬 

Do we want to skip `0` since you could end up with a number of `0123`?  `parseInt` still works but it doesn't skip the `0`, it just assumes the number is octal rather than decimal so can be a bit confusing.